gsoc_blog_setup.md

```diff
@@ -0,0 +1,131 @@
+Once you have been accepted as a GSoC student NumFOCUS expects you to blog about your progress every week. It's okay if you are stuck, post about that too :) You could use various platforms like wordpress, blogger, etc. but in our experience it's usually a pain to get formatting right in case of code snippets, screenshots. We suggest you use GitHub Pages and Jekyll to blog about your summer. 
+GitHub gives you free hosting and a free domain name (`github_username.github.io`) and [Jekyll](https://jekyllrb.com) is a blog friendly static site generator. This tutorial will help you setup your blog.
+
+#### Install Ruby
+For Debian or Ubuntu based users
+```
+$ sudo apt-get install ruby-full
+```
+Mac OS/OS X has ruby installed by default.
+
+For more information go through https://www.ruby-lang.org/en/documentation/installation/
+
+#### Install RVM and setup 
+```
+$ \curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
This is a bad security praxis. It's better to download the file and execute it separately. That way you a download that stops in the middle won't affect the user.